src/analyticsTracking/eventUtils.js

```diff
@@ -0,0 +1,119 @@
+import analyticsTrackingConstants from '../common/analyticsTrackingConstants';
+
+const EVENT_CONSTANTS = analyticsTrackingConstants;
+export const normalizeEventType = str =>
+  str
+    .trim()
+    .replace(/ /gi, '_')
+    .toUpperCase();
+
+export const mouseEvents = log => (eventProps = {}, eventType = null) => {
```

Comment:
It's hard to tell what the utility of these functions are without seeing them used in implementation, but it seems that they just are aliases for `log()` with a given event type? What's wrong with simply doing `log(EVENT_CONSTANTS.PAGE_VIEW)`?

Reply:
🔥